Duplicati/Library/Backend/S3/S3Backend.cs

```diff
@@ -392,7 +392,9 @@ public IList<ICommandLineArgument> SupportedCommands
 
                 var normal = new ICommandLineArgument[] {
                     new CommandLineArgument("aws_secret_access_key", CommandLineArgument.ArgumentType.Password, Strings.S3Backend.AMZKeyDescriptionShort, Strings.S3Backend.AMZKeyDescriptionLong, null, new string[] {"auth-password"}, null),

```

Comment:
For the old underscored options, can we use the following `CommandLineArgument` constructor that marks the options as deprecated?

https://github.com/duplicati/duplicati/blob/67aaab15e3a8638806a50cf23b5a085c826a338a/Duplicati/Library/Interface/CommandLineArgument.cs#L258-L263

Reply:
Shure. I added a new commit change. Could you review?